packages/autodiscovery/phpunit.xml

```diff
@@ -1,9 +1,7 @@
-<phpunit
-    bootstrap="vendor/autoload.php"
-    colors="true"
-    verbose="true"
->
-    <testsuite name="all">
-        <directory>tests</directory>
-    </testsuite>
+<?xml version="1.0"?>
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" bootstrap="vendor/autoload.php" colors="true" verbose="true" xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.3/phpunit.xsd">
+  <coverage/>
+  <testsuite name="all">
+    <directory>tests</directory>
+  </testsuite>
```

Comment:
Be sure to have `.editorconfig` plugin enabled in PHPStorm, so the spaces are not broken here.

Also keep original 1 config = 1 line syntax, so it's easier to review and read.

Reply:
Sorry, but this was auto generated by PHPUnit. I saw that it wanted to migrate the configuration. So I ran the command.

Do you think it's worth doing it?

Reply:
I reverted it as it didn't solve the color problem anyway.